src/trigger-circleci-workflows.ts

```diff
@@ -31,12 +31,12 @@ async function loadConfig(context: probot.Context): Promise<Config | {}> {
   const repoKey = utils.repoKey(context);
   let configObj = repoMap.get(repoKey);
   if (configObj === undefined) {
-    context.log.debug(`Grabbing config for ${repoKey}`, 'loadConfig');
+    context.log.info(`Grabbing config for ${repoKey}`, 'loadConfig');
```

Comment:
probot logger uses pino, looks like concat two strings no longer works in recent versions

https://github.com/pinojs/pino/blob/master/docs/api.md#interpolationvalues-any

```
logger.info('hello', 'world')
// {"level":30,"time":1531257618044,"msg":"hello","pid":55956,"hostname":"x"}
// world is missing
```

The function signature is like https://github.com/pinojs/pino/blob/master/docs/api.md#loggerinfomergingobject-message-interpolationvalues

Reply:
oh interesting... should I change all these in this PR? or in a followup?

Reply:
the same pattern is used in `src/auto-cc-bot.ts` and `src/auto-label-bot.ts` as well

Reply:
seems like a good change to include in this pr to make the logging right - e.g. just move two strings to one string. The other format like `context.log.info({}, string)` is good

Reply:
makes sense, I'll do that